p/api/greader.php

```diff
@@ -917,6 +921,12 @@ function markAllAsRead($streamId, $olderThanId) {
 if ($user !== '') {
 	FreshRSS_Context::$user_conf = get_user_configuration($user);
 	Minz_Translate::init(FreshRSS_Context::$user_conf->language);
+
+	Minz_ExtensionManager::init();
```

Comment:
Unless it makes a functional difference I think I'd just stick this on the line where it goes alphabetically (and in any case without newlines).

Some other part of the code seems to agree with me:
https://github.com/FreshRSS/FreshRSS/blob/03ddcf0772f77f8e67edcc43a71cb8d01c18a3ff/p/api/pshb.php#L119-L120

Reply:
Like this ?

```
	FreshRSS_Context::$user_conf = get_user_configuration($user);
	Minz_ExtensionManager::init();
	Minz_Translate::init(FreshRSS_Context::$user_conf->language);
	if (FreshRSS_Context::$user_conf != null) {
		Minz_ExtensionManager::enableByList(FreshRSS_Context::$user_conf->extensions_enabled);
	}
```

Reply:
Personally I'd stick in a newline before the if statement but something more like that, yes. ;-)